### PR TITLE
New banner design

### PR DESCRIPTION
**Pull request by vanitabarrett**
Trello: https://trello.com/c/DpCsD8yF/96-redesign-header-to-strengthen-stay-at-home-message

## What

- Grey background for additional information section
- Switch to using an SVG with fallback PNG (to mirror landing page)

## Before
Screenshot 2020-03-27 at 17 40 05

## After
Screenshot 2020-03-27 at 17 40 55

### CODE REVIEW

File: app/assets/stylesheets/helpers/_header.scss [hunk 1]
```diff
@@ -473,13 +473,14 @@
 }
 
 $covid-green: #7eff8e;
+$covid-grey: #262828;
```
Comment (andysellick) [inline app/assets/stylesheets/helpers/_header.scss]: I've just done the exact same thing in my PR 😁

File: app/assets/stylesheets/helpers/_header.scss [hunk 2]
```diff
@@ -497,81 +498,81 @@ $covid-green: #7eff8e;
...
+.global-bar__dismiss--show {
+  display: inline-block;
+}
 
-  .global-bar-additional {
-    display: none;
-    position: relative;
-    margin-top: govuk-spacing(4);
-    margin-bottom: govuk-spacing(4);
+.gem-c-govspeak .global-bar__heading {
```
Comment (andysellick) [inline app/assets/stylesheets/helpers/_header.scss]: Is it possible to not use a gem class outside of the gem? (I'm having a thing about isolation at the moment, pun not intended)

Comment (vanitabarrett) [reply app/assets/stylesheets/helpers/_header.scss]: @andysellick The govspeak styles were overriding the `.global-bar__heading` styles without this - I had to add `.gem-c-govspeak` to make it specific enough 😞

Comment (andysellick) [reply app/assets/stylesheets/helpers/_header.scss]: Okay, fair enough.

**Pull request merged**